airbyte-integrations/connectors/destination-snowflake/gradle.properties

```diff
@@ -1,4 +1,3 @@
 # currently limit the number of parallel threads until further investigation into the issues \
 # where Snowflake will fail to login using config credentials
-numberThreads=4
-parallelExecutionsPerThread=6
+testExecutionConcurrency=4
```

Comment:
This _seems_ OK in practice, despite it being a reduction.

Reply:
those tests will _definitely_ be IO bound. I'd bump that up to 50...

Reply:
there's some limit (either in the CI runner networking, or in the snowflake warehouse) that we run into beyond a certain parallelism limit. The right number might be somewhere in between :P

Reply:
we probably sbhould look at https://docs.snowflake.com/en/sql-reference/parameters#max-concurrency-level and increase that number on the snowflake side, if we're too low there. Doesn't change the costs of running queries